provisioning/service/src/Auth/SharedAccessSignature.cs

```diff
@@ -33,51 +29,23 @@ private SharedAccessSignature(string shareAccessSignatureName, DateTime expiresO
                 throw new UnauthorizedAccessException("The specified SAS token is expired");
             }
 
-            _shareAccessSignatureName = shareAccessSignatureName;
-            _signature = signature;
-            _audience = WebUtility.UrlDecode(encodedAudience);
+            ShareAccessSignatureName = shareAccessSignatureName;
+            Signature = signature;
+            Audience = WebUtility.UrlDecode(encodedAudience);
             _encodedAudience = encodedAudience;
```

Comment:
How are these being referenced/used? There is no get/set right?

Reply:
Is there a particular field you're asking about?
`_encodedAudience` in particular is being used while computing the signature (https://github.com/Azure/azure-iot-sdk-csharp/blob/abmisr/serviceInternal/provisioning/service/src/Auth/SharedAccessSignature.cs#L164)

Reply:
Fields which do not have getter/setters

Reply:
I see. Looked a bit weird on first glance since others are public now and this one is not.